src/shared/proto/robot_component_status.proto

```diff
@@ -0,0 +1,132 @@
+syntax = "proto2";
+
+/* Data about the status of the break beam */
+message BreakBeamStatus
+{
+    // Whether or not the ball is interrupting the robot’s laser beam.
+    required bool ball_in_beam = 1;
+
+    // The reading of the robot’s laser sensor.
+    required float break_beam_reading = 2;
+}
+
+/* Data about the status of the chipper and kicker */
+message ChipperKickerStatus
+{
+    /* Frankie Robot Fields */
+    // Indicates the time elapsed since chipper was last fired in ms
+    // UINT64_MAX indicates chipper was never fired
+    optional uint64 time_since_chipper_fired = 2;
+
+    // Indicates the time elapsed since kicker was last fired in ms
+    // UINT64_MAX indicates kicker was never fired
+    optional uint64 time_since_kicker_fired = 3;
+
+    /* 2014 Robot Fields */
+    // Indicates when autokick has been fired.
+    optional bool autokick_fired = 8;
```

Comment:
thoughts on whether we want `time_since_chipper_fired` and  `time_since_kicker_fired` instead of `autokick_fired` for the new robots?

Reply:
I don't think it's useful so would rather remove it.

Reply:
JK I have changed my mind, I actually think the `time_since_fired...` values are better than `autokick_fired`. Assuming we use these to help determine when robots have kicked the ball (for things like when an Action is done), the `time_since_fired` values are better because we can report something as fired if we receive any message with the `time_since` value being less than some threshold, indicating the chicker was fired in the last `n` seconds. This makes it easy to tune on the software side.

This is better because if we just had `autokick_fired`, we don't really know how long after the chicker fired that bool will be true. If we miss a few messages or the bool isn't true for very long, we may "miss" the chicker firing.

tl;dr I say keep the `time_since` values and remove `autokick_fired`

Reply:
done